### PR TITLE
meletrix-udev-rules: init at 0-unstable-2023-10-20

### PR DESCRIPTION
**Pull request by Scrumplex**
## Description of changes

These udev rules are helpful for configuring Zoom65 (and other Meletrix) keyboards using utilities like VIA.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/by-name/me/meletrix-udev-rules/package.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  stdenvNoCC,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "meletrix-udev-rules";
+  version = "unstable-2023-09-06";
```
Comment (FliegendeWurst) [inline pkgs/by-name/me/meletrix-udev-rules/package.nix:7]: There were some changes a few days ago (20th october)

**Review by FliegendeWurst — CHANGES_REQUESTED** [review]

**Review by FliegendeWurst — APPROVED** [review]

**Comment by pbsds** [comment]
This strikes me as a nixos-hardware or nixos config thing, not a package

**Comment by Scrumplex** [comment]
There are several `udev-rules` packages in Nixpkgs: https://search.nixos.org/packages?channel=unstable&query=udev-rules

File: pkgs/by-name/me/meletrix-udev-rules/package.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  stdenvNoCC,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "meletrix-udev-rules";
+  version = "unstable-2023-10-20";
```
Comment (pbsds) [inline pkgs/by-name/me/meletrix-udev-rules/package.nix:7]:
```suggestion
  version = "0-unstable-2023-10-20";
```

**Review by pbsds — COMMENTED** [review]
Oh TIL, thanks for pointing it out :)

Ofborg builds didn't trigger, likely due to the commit title. Please reword it to `meletrix-udev-rules: init at 0-unstable-2023-10-20`

**Comment by pbsds** [comment]
I'll reword it with a squash then

@ofborg build meletrix-udev-rules meletrix-udev-rules.passthru.tests

**Comment by Scrumplex** [comment]
I totally missed that the PR title and commit message didn't have the same date :sweat_smile:

**Comment by pbsds** [comment]
There is also the attrpath mismatch

**Comment by pbsds** [comment]
squashed!
